### PR TITLE
Created Rysgaard's Corona Special drink

### PR DESCRIPTION
**Pull request by Crowton**

### CODE REVIEW

File: src/recipes/rysgaardsCoronaSpecial.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "Mix the whiskey, soda and Old Danish",
+        "Observe the discusting color. Fix with the sauce coloring",
+        "Stir the drink counterclockwise until you feel better inside",
+        "If there is too much alcohol in the drink, you can always cry in it, to lower the percentage"
+    ],
+    "image": "rysgaardSpecial.jpg",
+    "keywords": [
+        "Whiskey",
+        "Corona",
+        "Old Danish"
```
Comment (alfg) [inline src/recipes/rysgaardsCoronaSpecial.json:37]: lowercase please

File: src/recipes/rysgaardsCoronaSpecial.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Rysgaard's Corona Special",
+    "description": "A drink for drinking alone in your appartment when quarentined.",
+    "github": "https://github.com/Crowton",
```
Comment (alfg) [inline src/recipes/rysgaardsCoronaSpecial.json:4]:
```suggestion
    "github": "Crowton",
```

File: src/recipes/rysgaardsCoronaSpecial.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Rysgaard's Corona Special",
+    "description": "A drink for drinking alone in your appartment when quarentined.",
```
Comment (alfg) [inline src/recipes/rysgaardsCoronaSpecial.json:3]:
```suggestion
    "description": "A drink for drinking alone in your apartment when quarantined.",
```

File: src/recipes/rysgaardsCoronaSpecial.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            "quantity": "3",
+            "measure": "drops",
+            "ingredient": "Dark coloring for sauce"
+        }
+    ],
+    "directions": [
+        "Mix the whiskey, soda and Old Danish",
+        "Observe the discusting color. Fix with the sauce coloring",
+        "Stir the drink counterclockwise until you feel better inside",
+        "If there is too much alcohol in the drink, you can always cry in it, to lower the percentage"
```
Comment (alfg) [inline src/recipes/rysgaardsCoronaSpecial.json:31]: 😆

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, can you also use lowercase and hyphens in the filenames only? The JSON and image file should match.

```
rysgaards-corona-special.jpg
rysgaards-corona-special.json
```

Thanks.

**Comment by Crowton** [comment]
> Hi, can you also use lowercase and hyphens in the filenames only? The JSON and image file should match.
> 
> ```
> rysgaards-corona-special.jpg
> rysgaards-corona-special.json
> ```
> 
> Thanks.

Sure

File: src/recipes/rysgaards-corona-special.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+            "ingredient": "Dark coloring for sauce"
+        }
+    ],
+    "directions": [
+        "Mix the whiskey, soda and Old Danish",
+        "Observe the discusting color. Fix with the sauce coloring",
+        "Stir the drink counterclockwise until you feel better inside",
+        "If there is too much alcohol in the drink, you can always cry in it, to lower the percentage"
+    ],
+    "image": "rysgaardSpecial.jpg",
```
Comment (alfg) [inline src/recipes/rysgaards-corona-special.json:33]:
```suggestion
    "image": "rysgaards-corona-special.jpg",
```

**Comment by alfg** [comment]
Thanks!

**Pull request merged**